sdk/keyvault/keyvault-keys/package.json

```diff
@@ -62,11 +65,11 @@
     "build:minify": "uglifyjs -c -m --comments --source-map \"content='./dist/index.js.map'\" -o ./dist/index.min.js ./dist/index.js 2>&1",
     "build:test": "npm run build",
     "build": "tsc && rollup -c rollup.config.js 2>&1",
-    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"tests/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist esm test-dist typings *.tgz *.log",
-    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"tests/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
+    "integration-test:node": "ts-mocha -p tests/tsconfig.test.json tests/*.ts",
```

Comment:
This test does not generate test results that the pipeline can run. This is accomplished in other packages with parameters like: 
```
--reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-
```

See also: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/package.json#L49